<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YearCalendar</class>
 <widget class="QWidget" name="YearCalendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Year View Calendar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QCalendarWidget" name="calendarApril"/>
   </item>
   <item row="1" column="1">
    <widget class="QCalendarWidget" name="calendarFebruary"/>
   </item>
   <item row="2" column="1">
    <widget class="QCalendarWidget" name="calendarJune"/>
   </item>
   <item row="3" column="1">
    <widget class="QCalendarWidget" name="calendarOctober"/>
   </item>
   <item row="3" column="3">
    <widget class="QCalendarWidget" name="calendarDecember"/>
   </item>
   <item row="3" column="2">
    <widget class="QCalendarWidget" name="calendarNovember"/>
   </item>
   <item row="1" column="2">
    <widget class="QCalendarWidget" name="calendarMarch"/>
   </item>
   <item row="1" column="0">
    <widget class="QCalendarWidget" name="calendarJanuary"/>
   </item>
   <item row="2" column="0">
    <widget class="QCalendarWidget" name="calendarMay"/>
   </item>
   <item row="3" column="0">
    <widget class="QCalendarWidget" name="calendarSeptember"/>
   </item>
   <item row="2" column="3">
    <widget class="QCalendarWidget" name="calendarAugust"/>
   </item>
   <item row="2" column="2">
    <widget class="QCalendarWidget" name="calendarJuly"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
